java-frontend/src/main/java/org/sonar/plugins/java/api/tree/Tree.java

```diff
@@ -547,6 +547,11 @@
      */
     STRING_LITERAL(LiteralTree.class),
 
+    /**
+     * {@link LiteralTree}
+     */
+    TEXT_BLOCK(LiteralTree.class),
```

Comment:
You might want to modify the LiteralTree class as well to mark in the comments that there is also this kind to consider.